cmd/README.md

```diff
@@ -215,6 +220,37 @@ This will store data in the same directory as your app, in a hidden folder calle
 
 Of course these options can be used in combination with one another.
 
+### Networking
+
+`hc run` uses mock networking by default and therefore can't talk to any other nodes.  If you want to test multiple nodes you will need to install the [n3h](https://github.com/holochain/n3h) networking component (following the instructions on the readme there).  Once you have installed it then you can simply fire up your first node while setting the HC_N3H_PATH environment variable to the path where you installed it.  If n3h was installed properly you should see something like this:
+
+``` shell
+$ HC_N3H_PATH=/home/eric/holochain/n3h hc run
```

Comment:
Don't you have to add `--networked` ? is the default for it be enabled or not? 

I can't see in the code a default for the `networked` variable

Reply:
you don't I just made it work with the presence of the environment variable.  I didn't even add the --networked flag for now.

Reply:
@zippy this is the code for a `--networked` option
https://github.com/holochain/holochain-rust/pull/826/files#diff-3f97b1cf8c268f48439642916602b667R113

Reply:
well, would you look at that!  I guess Julian did that before I started working on it.  You still have to set the ENV vars for now though... hopefully in the future that will go away.